app/src/pages/[resourceType]/[id].tsx

```diff
@@ -73,47 +110,72 @@ export default function ResourceIDPage({ jsonData }: InferGetServerSidePropsType
             <Text size="xl" color="gray">
               {jsonData.resourceType}/{jsonData.id}
             </Text>
-            <Button w={240} loading={draftMutation.isLoading} onClick={createDraftOfArtifact}>
-              Create Draft of {jsonData.resourceType}
-            </Button>
+            <Group>
+              <Button
+                w={240}
+                id="btn"
+                loading={isFetching}
+                loaderPosition="center"
+                onClick={() => {
+                  refetch();
+                }}
+              >
+                Get Data Requirements
+              </Button>
+              <Button w={240} loading={draftMutation.isLoading} onClick={createDraftOfArtifact}>
+                Create Draft of {jsonData.resourceType}
+              </Button>
+            </Group>
           </Group>
         </div>
         <Divider my="sm" pb={6} />
-        <Tabs variant="outline" defaultValue="json">
-          <Tabs.List>
-            <Tabs.Tab value="json">JSON</Tabs.Tab>
-            {decodedElm != null && <Tabs.Tab value="elm">ELM</Tabs.Tab>}
-            {decodedCql != null && <Tabs.Tab value="cql">CQL</Tabs.Tab>}
-            {jsonData.text && <Tabs.Tab value="narrative">Narrative</Tabs.Tab>}
-          </Tabs.List>
-          <Tabs.Panel value="json" pt="xs">
-            <Prism language="json" colorScheme="light">
-              {JSON.stringify(jsonData, null, 2)}
-            </Prism>
-          </Tabs.Panel>
-          {decodedCql != null && (
-            <Tabs.Panel value="cql" pt="xs">
-              {/* eslint-disable  @typescript-eslint/no-explicit-any */}
-              <Prism language={'cql' as any} colorScheme="light">
-                {/* eslint-enable  @typescript-eslint/no-explicit-any */}
-                {decodedCql}
-              </Prism>
-            </Tabs.Panel>
-          )}
-          {decodedElm != null && (
-            <Tabs.Panel value="elm" pt="xs">
+        <div>
```

Comment:
Now that I'm taking a closer look at the code diff in this file, I notice that it looks like a lot has changed with the tab code when really it's just that we moved all the tab code inside of a `<div>`. I don't think we need to wrap this all in a div (I didn't get any errors and none of the functionality changed) so I think we can remove the `<div>` tag on this line and on line 178